client/src/Footer.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import { Container, Header, List, Grid, Segment } from "semantic-ui-react";
+
+export const Footer = () => (
+  <Segment inverted vertical style={{ padding: "1em 0em", marginTop: "6em" }}>
+    <Container>
+      <Grid divided inverted stackable>
+        <Grid.Row>
+          <Grid.Column width={4}>
+            <Header inverted as="h4" content="Team" />
+            <List link inverted>
+              <List.Item as="a">Adnan Oglah</List.Item>
```

Comment:
Can you remove `as="a"` as it doesn't need to be a link.

Reply:
I like it as it highlights the name when you hover on it:)

Reply:
Ah okay cool, let's keep it like this then :)